extensions/src/ACRE2Arma/arma2ts/simplepipe_win32.cpp

```diff
@@ -131,7 +131,7 @@ bool PipeWin::OpenServer(HANDLE pipe, bool connect) {
 
 bool PipeWin::Write(const void* buf, size_t sz) {
     DWORD written = 0;
-    return (TRUE == ::WriteFile(pipe_, buf, sz, &written, NULL));
+    return (true == ::WriteFile(pipe_, buf, sz, &written, NULL));
```

Comment:
WriteFile returns BOOL, therefore it should be compared against TRUE
```suggestion
    return (TRUE == ::WriteFile(pipe_, buf, sz, &written, NULL));
```